lib/thrift/servers/binary_framed/acceptor_supervisor.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Thrift.Servers.BinaryFramed.AcceptorSupervisor do
+  @moduledoc """
+  A supervisor for acceptor processes.
+  """
+
+  alias Thrift.Servers.BinaryFramed.Acceptor
+  use Supervisor
+
+  def start_link(socket, server_module, handler_module, count) do
+    Supervisor.start_link(__MODULE__, [socket, server_module, handler_module, count])
+  end
+
+  def init([socket, server_module, handler_module, count]) do
+    children = for id <- (1..count) do
+      worker_id = "#{inspect handler_module}_acceptor_#{id}"
+      worker(Acceptor, [socket, server_module, handler_module], id: worker_id)
+    end
+    supervise(children, strategy: :one_for_one)
```

Comment:
The workers are permanent and the max restarts/max seconds will be the default 3/5. Given that the workers exit when their sockets close if a client machine goes down it will likely take down the server. I am not sure it is possible to have good fault tolerance with this simple design.